test/mercury/base-uri-pattern.test.ts

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
SHould try breaking your code some more here.  Do things like double `/`between segments.